### ISSUE TITLE
[CoE Starter Kit - QUESTION] Partial Audit Logs

### ISSUE THREAD

**Issue by TigerJ** [microsoft/coe-starter-kit#2349]
### What is your question?

I am currently configuring a COE and only receiving audit logs for the account I am setting up the COE with even though there should be many more logs for app launches from other users etc. Is there a common resolution to this or a place to start looking into why it is happening?

I am not receiving any errors or warnings, log data is populating, but it is not the volume it should be.

### What solution are you experiencing the issue with?

Audit Log

### What solution version are you using?

2.6

### What app or flow are you having the issue with?

[Child] Admin | Sync Logs

**Comment by Jenefer-Monroe** [comment]
There is no filter on user in place, including based on permissions. So it seems unlikely that you would only see your own launches in the table. 
We do have issues with partial returns of data based on the volume of data being too large. But we send an error message for this case to the admin email address you have in that environment variable. 
Please see step 11 of the [Set up the Power Automate flow](https://docs.microsoft.com/power-platform/guidance/coe/setup-auditlog#set-up-the-power-automate-flow) portion of the setup documentation for how to fix this volume issue.

If you are not seeing an error message in your email, check the value of the admin email [environment variable](https://docs.microsoft.com/power-platform/guidance/coe/faq#update-environment-variables), and check to see if you have unmanaged layers on the two flows in the audit log solution.

**Comment by TigerJ** [comment]
I re-performed the step 11 just in case, but it was done correctly when I reviewed how it was configured. 

Thank your for your help on this, I started with a Microsoft Support ticket: 2203300040004415
Because it involved the COE they said they could not assist when they called.

Below are some screenshots showing what we are seeing. After more research a colleague of mine was setting up a COE last may and filed a ticket when he was receiving no errors and no logs (not the same exact issue here) and Microsoft Support discovered the organization had a commercial tenant they were unaware of and a gcc tenant. He was not involved in the resolution as Microsoft Support took the issue over entirely. I have a feeling this may be a similar issue but we have no documentation on how to resolve. Again, this was a different circumstance, I am trying to provide as much detail as I can to resolve this. 

Thank you again.

![admin child flow](https://user-images.githubusercontent.com/1319169/161279479-58d78493-63bb-4941-9b5d-5b2fdd5f8460.png)
![Admin Sync Flow](https://user-images.githubusercontent.com/1319169/161279481-58612cf9-563d-41a0-b367-efd6dc6ebf48.png)
![admin-view audit logs](https://user-images.githubusercontent.com/1319169/161279485-ecc6618f-9beb-45d6-ab42-ab647dbc6178.png)

**Comment by TigerJ** [comment]
Thank you Jenefer, I have combed the suggestions above and everything appears correct. Are there any known issues with federated identities being used for setup? I was going to experiment with the account used for setup next to see if that does anything. Again, no errors and logs to populate, but only for the user I have been using to configure with. The power platform tenant does have many environments, apps and flows which are all properly loading in the admin view. Just the launch/audit detail is blank for everyone except our admin account, which does show launch info and audit details.

I appreciate your time. Thank you.

**Comment by Jenefer-Monroe** [comment]
if you browse to the audit log directly with the account, what does it see? 
https://compliance.microsoft.com/auditlogsearch?viewid=Search&rfr=scc_unifiedauditlog

**Comment by TigerJ** [comment]
Everything is disabled. Is there a different compliance url for GCC?

I believe the account was setup correctly for audit log access, and logs are populating which would confirm this to me.

Here is a screenshot:
![image](https://user-images.githubusercontent.com/1319169/161331244-e47068db-b9f0-4fa8-8856-a89d6818b909.png)

Thank you.

**Comment by Jenefer-Monroe** [comment]
I'm trying to see if you can view logs for other people there. I dont know if its a different URL for GCC
If you go to https://compliance.microsoft.com/homepage can you browse to Audit from there 

If that UX is disabled then there is something wrong

**Comment by TigerJ** [comment]
I do not see an area to view logs from the link above.
![image](https://user-images.githubusercontent.com/1319169/161333550-bfbbd99b-45df-4518-ba27-fb39e1230e20.png)

**Comment by Jenefer-Monroe** [comment]
I'm afraid I dont know either. You will need to do a little research to see how to find these logs in the UX for GCC. Right now it appears that your identity doesnt have correct access. Since you got to the logs page but did not have permissions to use it.

**Comment by TigerJ** [comment]
Thank you Jenefer!

I will dig around and see what I find over the weekend and early next week. I'll update this with any progress or further details if/when I get there. 

I appreciate your time and fast responses. Thanks again.

**Comment by Jenefer-Monroe** [comment]
Hi there. Checking in to see about this investigation. Thanks for using CoE!

**Comment by Jenefer-Monroe** [comment]
closing out as no further action for CoE starter kit team

**Comment by TigerJ** [comment]
Thanks @Jenefer-Monroe I just came by to update this and let you know it can be closed out. Not sure entirely what happened or what the fix entailed, after an additional support ticket with MS, I went into the child audit logs flow and hit save w/o changing anything, logs started coming in. We never had any errors though, and our admin account logs were coming in. super weird.

anyway, no issues now so thanks for closing!
